README.md

```diff
@@ -3,25 +3,24 @@
 [![Build Status](https://ci.centos.org/buildStatus/icon?job=devtools-fabric8-ui-npm-publish-build-master)](https://ci.centos.org/job/devtools-fabric8-ui-npm-publish-build-master)
 [![codecov](https://codecov.io/gh/fabric8io/fabric8-ui/branch/master/graph/badge.svg)](https://codecov.io/gh/fabric8io/fabric8-ui)
 
-## Before you start
+## Before you start: Pointing to a cluster
 
-### Backend API
+You need to setup your shell to point to the right cluster so that it can talk to the required back end services like KeyCloak, WIT, Forge, OpenShift etc.
 
-Make sure you set the URL to the services. For UI development, we recommend connecting to the dev environment API server. 
+We provide various sample environments out of the box which make it easier to get started. They are all located as bash scripts in `environments`. 
 
-To connect to the dev environment instances: 
-* `export FABRIC8_WIT_API_URL="http://api.prod-preview.openshift.io/api/"`
-* `export FABRIC8_RECOMMENDER_API_URL="http://api-bayesian.dev.rdu2c.fabric8.io/api/v1/"`
-* `export FABRIC8_FORGE_API_URL="https://forge.api.prod-preview.openshift.io"`
-* `export FABRIC8_REALM="fabric8"`
```

Comment:
wouldn't we use the pre-prod environment script though? The reason I removed that section is setting those few env vars is no longer enough really. Plus those env vars are not about using a local WIT - its about using pre-prod which doesn't really work any more anyway